app/ota_client.py

```diff
@@ -231,74 +231,93 @@ class _BaseInf:
         r"(?P<mode>\d+),(?P<uid>\d+),(?P<gid>\d+),(?P<left_over>.*)"
     )
 
+    __slots__ = ["mode", "uid", "gid", "_left"]
+
     @staticmethod
     def de_escape(s: str) -> str:
         return s.replace(r"'\''", r"'")
 
     def __init__(self, info: str):
-        match_res: re.Match = self._base_pattern.match(info.strip("\n"))
+        match_res: re.Match = self._base_pattern.match(info.strip())
```

Comment:
@Bodong-Yang This strip() strips not only \n but also (=space)?
This should be reverted?

Reply:
OK!

Reply:
> end with white space?

I think info.strip is not only `end with white space`. Is my understanding correct?

Reply:
Ah, I thought "a b c".strip() returns "abc", but it returns "a b c".
But " a b c ".strip() returns "a b c", so I think strip("\n") should be used.

Reply:
@obi-t4 Ah I mean it there any case that info string will need to preserved the ending white space? Path itself is wrapped with `'` so I think any white space at the end of info string is useless?

Reply:
Yes, you're right! strip() is fine!

Reply:
> Ah, I thought "a b c".strip() returns "abc", but it returns "a b c". But " a b c ".strip() returns "a b c", so I think strip("\n") should be used.

Yes, `strip` only strip alway the prefix/suffix ` ` or `\n`